community-operators/federation/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,192 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation.v0.0.5
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Integration & Delivery"
+    description: "Kubernetes Federation namespace-scoped installation"
+    certified: false
+    containerImage: "quay.io/openshift/origin-federation-controller"
```

Comment:
The image should match the one used in the deployment section

Reply:
It does. Do you mean that the tag is expected to be present?

Examples would really help for this in the link you sent - would you open an issue to add examples for the required fields?